bazel/experiments.bzl

```diff
@@ -20,6 +20,9 @@ EXPERIMENTS = {
     "dbg": {
     },
     "off": {
+        "core_end2end_tests": [
+            "new_hpack_huffman_decoder",
```

Comment:
are tests also experiments?

Reply:
tests get enabled by experiments: we automatically run some subset of the test suite with each experiment in both states (on and off)